applications/tari_walletd/web_ui/src/routes/Transactions/TransactionDetails.tsx

```
@@ -37,11 +37,11 @@ import Inputs from "./Inputs";
 import Signers from "./Signers";
 import ExecutionResults from "./ExecutionResults";
 import FeeReceipt from "./FeeReceipt";
-import StatusChip from "../../Components/StatusChip";
+import StatusChip from "@components/StatusChip";
 import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
 import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
-import Loading from "../../Components/Loading";
-import Error from "../../Components/Error";
+import Loading from "@components/Loading";
+import Error from "@components/Error";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)